drivers/clock_control/clock_control_ast10x0.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2022 ASPEED Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT aspeed_ast10x0_clock
+#include <errno.h>
+#include <zephyr/dt-bindings/clock/ast10x0_clock.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/syscon.h>
+
+#define LOG_LEVEL CONFIG_CLOCK_CONTROL_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(clock_control_ast10x0);
+
+#define HPLL_FREQ			1000000000U
```

Comment:
I'm guessing this is in HZ.  So maybe use `MHZ()` macro from `include/zephyr/sys/util.h` to make it a bit more readable.